lua/lspconfig/server_configurations/please.lua

```diff
@@ -0,0 +1,15 @@
+local util = require 'lspconfig.util'
+
+return {
+  default_config = {
+    cmd = { 'plz', 'tool', 'lps' },
+    filetypes = { 'bzl' },
+    root_dir = util.root_pattern '.plzconfig',
+    single_file_support = true,
+  },
+  docs = {
+    description = [[
+`plz` will automatically install the LSP for you on first run
```

Comment:
Probably should flesh out the description a bit more, check out pyright. At minimum is nice to have a GH url